bftengine/src/communication/TlsTCPCommunication.cpp

```diff
@@ -558,32 +560,25 @@ class AsyncTlsConnection : public enable_shared_from_this<AsyncTlsConnection> {
 
   void read_msg_async_completed(const boost::system::error_code &ec,
                                 size_t bytesRead) {
-    // (IG): patch, dont do it, need to fix multithreading
-    LOG_TRACE("enter, node " << _selfId << ", dest: " << _destId);
+    LOG_TRACE(_logger, "enter, node " << _selfId << ", dest: " << _destId);
```

Comment:
What does this trace tell you? I see `enter, node` or `enter node` in multiple places. Are the function names already traced? If not, perhaps we want to add them, rather than enter node.